docs/DeveloperGuide.adoc

```diff
@@ -1482,6 +1482,54 @@ These instructions only provide a starting point for testers to work on; testers
 .. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size) +
    Expected: Similar to previous.
 
+=== Sorting problems
+
+.. Test case: `sortprob m/name` +
+   Expected: The current problem list is sorted with respect to names in ascending order.
+.. Test case: `sortprob m/difficulty ord/descend` +
+   Expected: The current problem list is sorted with respect to difficulty in descending order.
+.. Test case: `sortprob m/diff` +
+   Expected: The current problem list remains unchanged and an error is thrown to inform the user that the method name is incorrect.
+.. Test case: `sortprob m/diff m/name` +
+   Expected: The current problem list is sorted with respect to name in ascending order.
+.. Test case: `sortprob m/name m/diff` +
+   Expected: The current problem list remains unchanged and an error is thrown to inform the user that the method name is incorrect.
```

Comment:
Can change to "...the method name 'diff' is incorrect as the parser only takes the last parameter when there are duplicates"